.github/workflows/ci.yml

```diff
@@ -5,6 +5,7 @@ on:
     branches: [ master ]
   pull_request:
     branches: [ master ]
+  workflow_call:
```

Comment:
This is just to support `uses:` above?

Reply:
Yes, you need to add that apparently to make it so that you can call the workflow from another one. It's called a reusable workflow: https://docs.github.com/en/actions/using-workflows/reusing-workflows